plugins/destination/s3/client/spec.go

```diff
@@ -12,13 +12,19 @@ const (
 )
 
 type Spec struct {
-	Bucket   string     `json:"bucket,omitempty"`
-	Path     string     `json:"path,omitempty"`
-	Format   FormatType `json:"format,omitempty"`
-	NoRotate bool       `json:"no_rotate,omitempty"`
+	Bucket         string     `json:"bucket,omitempty"`
+	Path           string     `json:"path,omitempty"`
+	Format         FormatType `json:"format,omitempty"`
+	IncludeHeaders bool       `json:"include_headers,omitempty"`
+	Delimiter      rune       `json:"delimiter,omitempty"`
```

Comment:
Should this be a `string` instead of a `rune`? The docs say `string` as the type but explain it's a character.
Do we mind letting people use `, ` and not only `,`?

Reply:
@erezrokah We can use a string here, but we'd always have to use only the first rune, because the Go CSV library only takes a rune https://pkg.go.dev/encoding/csv#Writer

Reply:
Ah, so lets keep `rune` so if someone uses a string reading the config fails. Maybe update the docs to say `char` instead of string?